Source/RhetosCli/ApplicationDeployment.cs

```diff
@@ -0,0 +1,212 @@
+﻿/*
+    Copyright (C) 2014 Omega software d.o.o.
+
+    This file is part of Rhetos.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as
+    published by the Free Software Foundation, either version 3 of the
+    License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+using Autofac;
+using Rhetos;
+using Rhetos.Deployment;
+using Rhetos.Dom;
+using Rhetos.Extensibility;
+using Rhetos.Logging;
+using Rhetos.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Rhetos
+{
+    public class ApplicationDeployment

```

Comment:
I think we should try to reuse features between DeployPackages.exe and Rhetos.exe, with as little redundancy in code as possible, as least for now. That will help us to have clear insight into differences in behavior, and implement the differences explicitly stated in source code.